teuthology/task/install/__init__.py

```diff
@@ -607,6 +607,7 @@ def task(ctx, config):
                 sha1=config.get('sha1'),
                 tag=config.get('tag'),
                 wait_for_package=config.get('wait_for_package', False),
+                shaman=config.get('shaman', {})
```

Comment:
Nit: perhaps treat this more like the `repos` key below:

```
if 'shaman' in config:
    nested_config['shaman'] = config['shaman']
```

Reply:
Done.